integration-cli/docker_cli_cp_test.go

```diff
@@ -26,107 +27,77 @@ const (
 // Ensure that an all-local path case returns an error.
 func (s *DockerSuite) TestCpLocalOnly(c *check.C) {
 	err := runDockerCp(c, "foo", "bar")
-	if err == nil {
-		c.Fatal("expected failure, got success")
-	}
+	c.Assert(err, checker.NotNil)
 
-	if !strings.Contains(err.Error(), "must specify at least one container source") {
-		c.Fatalf("unexpected output: %s", err.Error())
-	}
+	c.Assert(err.Error(), checker.Contains, "must specify at least one container source")
 }
 
 // Test for #5656
 // Check that garbage paths don't escape the container's rootfs
 func (s *DockerSuite) TestCpGarbagePath(c *check.C) {
 	testRequires(c, DaemonIsLinux)
-	out, exitCode := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", "mkdir -p '"+cpTestPath+"' && echo -n '"+cpContainerContents+"' > "+cpFullPath)
-	if exitCode != 0 {
-		c.Fatal("failed to create a container", out)
-	}
+	out, _ := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", "mkdir -p '"+cpTestPath+"' && echo -n '"+cpContainerContents+"' > "+cpFullPath)
 
-	cleanedContainerID := strings.TrimSpace(out)
+	cID := strings.TrimSpace(out)
```

Comment:
In these files, I found four names : `cleanedContainerID`, `containerID`, `cID`, `id`. So I unified them to `cID`. Maybe `containerID` is more better.